README.md

```diff
@@ -70,21 +72,43 @@ Install docker and run SecretScanner on a container image using the following in
     docker run -it --rm --name=deepfence-secretscanner -v $(pwd):/home/deepfence/output -v /var/run/docker.sock:/var/run/docker.sock deepfenceio/secretscanning -local /home/deepfence/src/SecretScanner/test
     ```
 
+* Or run SecretScanner as a gRPC server:
+	```
+	docker run -it --rm --name=deepfence-secretscanner -v $(pwd):/home/deepfence/output -v /var/run/docker.sock:/var/run/docker.sock -v /tmp/sock:/tmp/sock deepfenceio -socket-path /tmp/sock/s.sock
```

Comment:
I think this command should change depending on the underlying docker sock. What will be the scenario when we use containerd ?

Reply:
PR https://github.com/deepfence/SecretScanner/pull/23 addresses the issue